beacon-chain/db/kv/blocks_test.go

```diff
@@ -304,9 +305,9 @@ func TestStore_Blocks_Retrieve_SlotRange(t *testing.T) {
 
 func TestStore_Blocks_Retrieve_Epoch(t *testing.T) {
 	db := setupDB(t)
-	slots := params.BeaconConfig().SlotsPerEpoch * 7
+	slots := params.BeaconConfig().SlotsPerEpoch.Mul(7)
```

Comment:
An overall comment: `SlotsPerEpoch` and similar config values should be numbers. The same goes for epochs.